var/spack/repos/builtin/packages/dla-future/package.py

```diff
@@ -124,6 +124,8 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
             depends_on("pika cuda_arch={0}".format(val), when="cuda_arch={0}".format(val))
             depends_on("umpire cuda_arch={0}".format(val), when="cuda_arch={0}".format(val))
 
+    patch("stdint.patch", when="@0.3.1 %gcc@13")
```

Comment:
Thanks for taking care of this. Would you mind updating this to
```suggestion
    patch(
        "https://github.com/eth-cscs/DLA-Future/pull/1063/commits/efc9c176a7a8c512b3f37d079dec8c25ac1b7389.patch?full_index=1",
        sha256="7f382c872d89f22da1ad499e85ffe9881cc7404c8465e42877a210a09382e2ea",
        when="@:0.3 %gcc@13:",
    )
```
The patch applies for all versions up to and including 0.3.1 and is required with gcc 13 and newer. The patch comes from https://github.com/eth-cscs/DLA-Future/pull/1063 which is the upstream fix for the same problem.

Reply:
maybe use https://github.com/eth-cscs/DLA-Future/commit/1f7993860ed4e6d0342c7ecef40c343f28a87049 directly instead of the PR?